RELEASE-NOTES.md

```diff
@@ -27,6 +27,8 @@
   * `snow snowpark deploy` uploads all artifacts created during build step. Dependencies zip is upload once to
      every Snowpark stage specified in project definition.
   * The changes are compatible with V1 projects definition though the result state (file layout) is different.
+  * `snow snowpark package` commands no longer fallback to Anaconda Channel metadata when fetching available packages info fails.
+* `ENABLE_PROJECT_DEFINITION_V2` feature flag is removed. Project definition V2 is always enabled.
```

Comment:
This is probably not necessary, I don't think we ever officially said that there's something like this.

Reply:
Removed